src/widgets/index/index.ts

```diff
@@ -403,6 +406,10 @@ const index = (props: IndexProps): Index => {
       // configuration of the widget is pushed in the URL. That's what we want to avoid.
       // https://github.com/algolia/instantsearch.js/pull/994/commits/4a672ae3fd78809e213de0368549ef12e9dc9454
       helper.on('change', ({ state }) => {
+        if (!instantSearchInstance.started) {
+          return;
+        }
```

Comment:
adding these lines will make a bunch of test fail related specifically to this behaviour before start. The tests in index seem to make sense to need .started, but the ones in RoutingManager seem like legit bug fixes

I've considered doing something like `_isSsrServer` (passing Vue's `this.$isServer` to instance), but not sure if that makes full sense.